### ISSUE TITLE
add fiat estimations

### ISSUE THREAD

**Issue by pciavald** [particl/partgui#568]
It would be very convenient (and almost mandatory when we launch the marketplace) to have a fiat estimate of PART amounts in the wallets.

I would see it fit : 
- in the Overview header
- in the Send balances and amounts
- in each transaction (transaction-table component)

CoinMarketCap is the price provider that would make the more sense, do we want to use something else ?

**Comment by pciavald** [comment]
@rynomster what do you think ?

**Comment by allienworks** [comment]
Absolutely, however I'd love to include more currencies then USD/EUR if possible. For example Coindesk's BPI is using [Open Exchange Rates](https://openexchangerates.org/) to convert prices to other currencies.

**Comment by zaSmilingIdiot** [comment]
aking requests directly to an API means leaking your connection to the service provider at least, and worse means reliance on a third party system built directly into the application (this latter point concerns cases when the 3rd party service shuts down, or is compromised, or sold, or changes access requirements, etc).

However, similar functionality exists within the particl-marketplace codebase currently (despite being turned off). Such an implementation, for privacy reasons, probably requires an implementation similar to an oracle service, with pricing information distributed via smsg messages. For example.
